src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -641,7 +652,6 @@ private List<EquipmentModificationInfos> execCreateLoad(NetworkStoreListener lis
                     .withSeverity(TypedValue.INFO_SEVERITY)
                     .build());
```

Comment:
Add this report for the load creation only in BUS_BREAKER ?
because, in NODE_BREAKER, a report is already added in the CreateFeederBay apply method

Reply:
done